tests/layer.test.js

```diff
@@ -22,24 +22,44 @@ test('modify Layer fields', () => {
     expect(layer.id).toBe('new-layer-id');
 });
 
-test('validate Layer: invalid values', () => {
+test('validate Layer: invalid name value', () => {
     let badNameLayer = new Layer('', 'id');
-    let badIDLayer = new Layer('name', '');
-    let badBothLayer = new Layer('', '');
-    let goodLayer = new Layer('good-name', 'good-id');
 
     expect(badNameLayer.validate()).toBe(false);
+});
+
+test('validate Layer: invalid id value', () => {
+    let badIDLayer = new Layer('name', '');
+
     expect(badIDLayer.validate()).toBe(false);
+});
+
+test('validate Layer: invalid values', () => {
```

Comment:
Can you specify which values are invalid here (name and id)

Reply:
They have been added.